config/routes.rb

```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  get 'errors/not_found', to: 'errors#not_found'
+
+  get 'errors/internal_server_error', to: 'errors#internal_server_error'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.